lib/tuya.js

```diff
@@ -47,6 +47,7 @@ function getDataValue(dataType, data) {
 }
 
 function convertDecimalValueTo4ByteHexArray(value) {
+    if(value % 1 !== 0) value = Math.round(Number(value))
```

Comment:
Better were to convert the float value e.g. 2.4 to Hex, didn't know exacliy how is possible